soh/soh/Enhancements/enemyrandomizer.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <stdint.h>
+
+typedef struct enemyEntry {
+    int16_t enemyId;
+    int16_t enemyParam;
+} enemyEntry;
+
+enum { RANDOMIZED_ENEMY_SPAWN_TABLE_SIZE = 47 };
```

Comment:
it's not clear why this is an enum, maybe just `#define` it?

Reply:
Also true, will change this.